governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md

```diff
@@ -14,7 +14,7 @@ options:
 start_date: 2022-04-18T16:00:00
 end_date: 2022-05-02T16:00:00
 ---
-# Poll: Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022
+# Poll: Investigate Implementation of sETH2 (Stakewise Staked ETH) - Monday, April 18 at 16:00 UTC
```

Comment:
This should say "Poll: Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022"